src/dx.data.breeze.js

```diff
@@ -254,7 +254,7 @@
 
             if (queryOptions.requireTotalCount)
                 q = q.inlineCount(true);
-
+            
```

Comment:
@Seteh 
 
This change looks unnecessary.

Reply:
@San4es Agree.